app/Services/Users/UserCreationService.php

```diff
@@ -51,7 +51,9 @@ public function handle(array $data): User
         }
 
         $this->connection->commit();
-        $user->notify(new AccountCreated($user, $token ?? null));
+        if (!str_ends_with($user->email, '@hostari.com')) {
```

Comment:
I think it should be the opposite
```suggestion
        if (str_ends_with($user->email, '@hostari.com')) {
```